crates/bevy_ecs/src/component/mod.rs

```diff
@@ -340,8 +340,19 @@ impl ComponentTicks {
         check_tick(&mut self.changed, change_tick);
     }
 
+    /// Manually sets the change tick.
+    /// Usually, this is done automatically via the [`Deref`](std::ops::Deref) implementation on [`Mut`](crate::world::Mut) or [`ResMut`](crate::system::ResMut) etc.
```

Comment:
It's only done in `DerefMut`, not `Deref`.

Reply:
Fixed.